<?xml version="1.0" encoding="UTF-8" ?>
<testsuite tests="1" failures="0" name="com.sy.test.OrderTest" time="0.014" errors="1" skipped="0">
  <properties>
    <property name="idea.version" value="2021.3.3"/>
    <property name="java.runtime.name" value="Java(TM) SE Runtime Environment"/>
    <property name="java.vm.version" value="17.0.4.1+1-LTS-2"/>
    <property name="sun.boot.library.path" value="D:\java-jdk\bin"/>
    <property name="maven.multiModuleProjectDirectory" value="D:\app\Admin_order_del"/>
    <property name="java.vm.vendor" value="Oracle Corporation"/>
    <property name="java.vendor.url" value="https://java.oracle.com/"/>
    <property name="guice.disable.misplaced.annotation.check" value="true"/>
    <property name="path.separator" value=";"/>
    <property name="java.vm.name" value="Java HotSpot(TM) 64-Bit Server VM"/>
    <property name="sun.os.patch.level" value=""/>
    <property name="user.script" value=""/>
    <property name="user.country" value="CN"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="D:\app\Admin_order_del"/>
    <property name="java.vm.compressedOopsMode" value="Zero based"/>
    <property name="forkCount" value="0"/>
    <property name="intellij.debug.agent" value="true"/>
    <property name="java.runtime.version" value="17.0.4.1+1-LTS-2"/>
    <property name="os.arch" value="amd64"/>
    <property name="java.io.tmpdir" value="C:\Users\SHIYUQ~1\AppData\Local\Temp\"/>
    <property name="line.separator" value="
"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="user.variant" value=""/>
    <property name="os.name" value="Windows 11"/>
    <property name="reuseForks" value="false"/>
    <property name="maven.ext.class.path" value="D:\IDEA\IntelliJ IDEA 2021.3.3\plugins\maven\lib\maven-event-listener.jar"/>
    <property name="classworlds.conf" value="D:\MAVEN\apache-maven-3.8.6\bin\m2.conf"/>
    <property name="sun.jnu.encoding" value="GBK"/>
    <property name="java.library.path" value="D:\java-jdk\bin;C:\WINDOWS\Sun\Java\bin;C:\WINDOWS\system32;C:\WINDOWS;D:\VMware\bin\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\dotnet\;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;D:\MAVEN\apache-maven-3.8.6\bin;D:\java-jdk\bin;D:\MySQL\bin;C:\Program Files\MySQL\MySQL Shell 8.0\bin\;C:\Users\shiyuqingchengse\AppData\Local\Microsoft\WindowsApps;C:\Users\shiyuqingchengse\.dotnet\tools;D:\IDEAL\IntelliJ IDEA 2021.2\bin;;;D:\Idea\IntelliJ IDEA Community Edition 2022.2.3\bin;;."/>
    <property name="jboss.modules.system.pkgs" value="com.intellij.rt"/>
    <property name="maven.conf" value="D:\MAVEN\apache-maven-3.8.6/conf"/>
    <property name="jdk.debug" value="release"/>
    <property name="java.class.version" value="61.0"/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="maven.repo.local" value="D:\MAVEN\apache-maven-3.8.6\mvn_repo"/>
    <property name="os.version" value="10.0"/>
    <property name="user.home" value="C:\Users\shiyuqingchengse"/>
    <property name="user.timezone" value="Asia/Shanghai"/>
    <property name="file.encoding" value="UTF-8"/>
    <property name="java.specification.version" value="17"/>
    <property name="user.name" value="shiyuqingchengse"/>
    <property name="java.class.path" value="D:\MAVEN\apache-maven-3.8.6\boot\plexus-classworlds-2.6.0.jar;D:\MAVEN\apache-maven-3.8.6\boot\plexus-classworlds.license;D:\IDEA\IntelliJ IDEA 2021.3.3\lib\idea_rt.jar"/>
    <property name="java.vm.specification.version" value="17"/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="sun.java.command" value="org.codehaus.classworlds.Launcher -Didea.version=2021.3.3 -s D:\MAVEN\apache-maven-3.8.6\conf\settings.xml -Dmaven.repo.local=D:\MAVEN\apache-maven-3.8.6\mvn_repo package -DforkCount=0 -DreuseForks=false"/>
    <property name="java.home" value="D:\java-jdk"/>
    <property name="user.language" value="zh"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="java.vm.info" value="mixed mode, sharing"/>
    <property name="java.version" value="17.0.4.1"/>
    <property name="native.encoding" value="GBK"/>
    <property name="java.vendor" value="Oracle Corporation"/>
    <property name="maven.home" value="D:\MAVEN\apache-maven-3.8.6"/>
    <property name="file.separator" value="\"/>
    <property name="java.version.date" value="2022-08-18"/>
    <property name="java.vendor.url.bug" value="https://bugreport.java.com/bugreport/"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="sun.cpu.isalist" value="amd64"/>
  </properties>
  <testcase classname="com.sy.test.OrderTest" name="testAdd" time="0.014">
    <error message="
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry &apos;1&apos; for key &apos;orderitem.PRIMARY&apos;
### The error may exist in com/sy/mapper/OrderitemMapper.xml
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### SQL: insert into orderitem(id,ordersid,productid,quantity)         values (?,?,?,?)
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry &apos;1&apos; for key &apos;orderitem.PRIMARY&apos;" type="org.apache.ibatis.exceptions.PersistenceException">org.apache.ibatis.exceptions.PersistenceException: 
### Error updating database.  Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry &apos;1&apos; for key &apos;orderitem.PRIMARY&apos;
### The error may exist in com/sy/mapper/OrderitemMapper.xml
### The error may involve defaultParameterMap
### The error occurred while setting parameters
### SQL: insert into orderitem(id,ordersid,productid,quantity)         values (?,?,?,?)
### Cause: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry &apos;1&apos; for key &apos;orderitem.PRIMARY&apos;
	at org.apache.ibatis.exceptions.ExceptionFactory.wrapException(ExceptionFactory.java:30)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:199)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.insert(DefaultSqlSession.java:184)
	at org.apache.ibatis.binding.MapperMethod.execute(MapperMethod.java:62)
	at org.apache.ibatis.binding.MapperProxy$PlainMethodInvoker.invoke(MapperProxy.java:152)
	at org.apache.ibatis.binding.MapperProxy.invoke(MapperProxy.java:85)
	at jdk.proxy2/jdk.proxy2.$Proxy8.add(Unknown Source)
	at com.sy.test.OrderTest.testAdd(OrderTest.java:42)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLIntegrityConstraintViolationException: Duplicate entry &apos;1&apos; for key &apos;orderitem.PRIMARY&apos;
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.Util.getInstance(Util.java:408)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3976)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3912)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2486)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1858)
	at com.mysql.jdbc.PreparedStatement.execute(PreparedStatement.java:1197)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.apache.ibatis.logging.jdbc.PreparedStatementLogger.invoke(PreparedStatementLogger.java:59)
	at jdk.proxy3/jdk.proxy3.$Proxy10.execute(Unknown Source)
	at org.apache.ibatis.executor.statement.PreparedStatementHandler.update(PreparedStatementHandler.java:47)
	at org.apache.ibatis.executor.statement.RoutingStatementHandler.update(RoutingStatementHandler.java:74)
	at org.apache.ibatis.executor.SimpleExecutor.doUpdate(SimpleExecutor.java:50)
	at org.apache.ibatis.executor.BaseExecutor.update(BaseExecutor.java:117)
	at org.apache.ibatis.executor.CachingExecutor.update(CachingExecutor.java:76)
	at org.apache.ibatis.session.defaults.DefaultSqlSession.update(DefaultSqlSession.java:197)
	... 38 more
</error>
    <system-out>[DEBUG]  &amp;#27;[36m[main]&amp;#27;[0;39m &amp;#27;[1;32mo.a.i.d.p.PooledDataSource&amp;#27;[0;39m - PooledDataSource forcefully closed/removed all connections. 
[DEBUG]  &amp;#27;[36m[main]&amp;#27;[0;39m &amp;#27;[1;32mo.a.i.d.p.PooledDataSource&amp;#27;[0;39m - PooledDataSource forcefully closed/removed all connections. 
[DEBUG]  &amp;#27;[36m[main]&amp;#27;[0;39m &amp;#27;[1;32mo.a.i.d.p.PooledDataSource&amp;#27;[0;39m - PooledDataSource forcefully closed/removed all connections. 
[DEBUG]  &amp;#27;[36m[main]&amp;#27;[0;39m &amp;#27;[1;32mo.a.i.d.p.PooledDataSource&amp;#27;[0;39m - PooledDataSource forcefully closed/removed all connections. 
[DEBUG]  &amp;#27;[36m[main]&amp;#27;[0;39m &amp;#27;[1;32mo.a.i.t.j.JdbcTransaction&amp;#27;[0;39m - Opening JDBC Connection 
[DEBUG]  &amp;#27;[36m[main]&amp;#27;[0;39m &amp;#27;[1;32mo.a.i.d.p.PooledDataSource&amp;#27;[0;39m - Created connection 310016558. 
[DEBUG]  &amp;#27;[36m[main]&amp;#27;[0;39m &amp;#27;[1;32mo.a.i.t.j.JdbcTransaction&amp;#27;[0;39m - Setting autocommit to false on JDBC Connection [com.mysql.jdbc.JDBC4Connection@127a7a2e] 
[DEBUG]  &amp;#27;[36m[main]&amp;#27;[0;39m &amp;#27;[1;32mc.s.m.O.add&amp;#27;[0;39m - ==&gt;  Preparing: insert into orderitem(id,ordersid,productid,quantity) values (?,?,?,?) 
[DEBUG]  &amp;#27;[36m[main]&amp;#27;[0;39m &amp;#27;[1;32mc.s.m.O.add&amp;#27;[0;39m - ==&gt; Parameters: 1(Integer), 2(Integer), 3(Integer), good(String) 
</system-out>
  </testcase>
</testsuite>